src/renderer/pages/toast/modules/keyboard.js

```
@@ -106,12 +95,9 @@ export function handleKeyDown(event) {
       // Use event.code to get physical key location instead of event.key
       const keyCode = event.code;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)